services/grafana-loki/0.48.4/defaults/cm.yaml

```diff
@@ -16,6 +16,8 @@ data:
           http_listen_port: 3100
           # info is a little too quiet.
           log_level: debug
+          grpc_server_max_recv_msg_size: 10485760
```

Comment:
should we add a comment that this value needs to be greater than the nginx client_max_body_size?  (right?)